### PR TITLE
feat(meta): add metrics for idle compactor

### PR DESCRIPTION
**Pull request by Little-Wallace**
Signed-off-by: Little-Wallace <bupt2013211450@gmail.com>

I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

To measure whether our system is busy, we need to know how many compactor could receive more task in time.

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

If your pull request contains user-facing changes, please specify the types of the changes, and create a release note. Otherwise, please feel free to remove this section.

### Types of user-facing changes

Please keep the types that apply to your changes, and remove those that do not apply.

* Installation and deployment 
* Connector (sources & sinks)
* SQL commands, functions, and operators
* RisingWave cluster configuration changes
* Other (please specify in the release note below)

### Release note

Please create a release note for your changes. In the release note, focus on the impact on users, and mention the environment or conditions where the impact may occur.

### CODE REVIEW

File: grafana/risingwave-dashboard.py [hunk 1]
```diff
@@ -223,7 +223,11 @@ def section_compaction(outer_panels):
                     "avg(storage_compact_task_pending_num) by(job, instance)", "compactor_task_split_count - {{job}} @ {{instance}}"
                 ),
             ]),
-
+            panels.timeseries_count("Idle Compactor Count", [
```
Comment (Li0k) [inline grafana/risingwave-dashboard.py:226]: can we calculate the percentage of task_num occupy, like `compacting_task_num / global_max_max_concurrent_task_number` to  measure the occupy of task_assign

**Review by Li0k — APPROVED** [review]
Rest LGTM

File: grafana/risingwave-dashboard.py (hunk 1 above)
Comment (Little-Wallace) [reply grafana/risingwave-dashboard.py:226]: Good Idea.

**Comment by Gun9niR** [comment]
Is this PR related to compactor scaling?

**Comment by Little-Wallace** [comment]
> Is this PR related to compactor scaling?

Yes

**Comment by xxchan** [comment]
Can this PR be merged or closed?

**Comment by xxchan** [comment]
Close as stale. Feel free to reopen.

**Pull request closed without merging**